src/App.js

```diff
@@ -0,0 +1,24 @@
+import { Route, Routes } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from './redux/Books/ConfigureStore';
```

Comment:
Kindly check your naming as it is throwing errors that prevent the project from running.

![image](https://user-images.githubusercontent.com/27709832/203822958-e675cd06-8063-46db-bbbf-9d8792c3aac3.png)

Reply:
ok fixed